trpcultivate_phenotypes/tests/src/Kernel/ServiceTermTest.php

```diff
@@ -176,35 +224,105 @@ public static function provideTermsForSaveTermConfigValuesMethod() {
   }
 
   /**
-   * Test Term Service getTermId() method.
+   * Test Term Service saveTermConfigValues() method.
    *
    * @param string $scenario
    *   A string, human-readable short descriptionn of the test scenario.
-   * @param string $input_term_identifier
-   *   A string, term identifier input.
+   * @param array $input_term
+   *   An array, term input with the following keys:
+   *     - 'name': the name of the term.
+   *     - 'cv': the cv vocabulary the term will be associated.
+   * @param string $term_identifier
+   *   A string, the term identifier of the module the term array will
+   *   be saved and mapped to.
    * @param array $expected
    *   An array of expected values, with the following keys.
-   *     - 'term_exists': boolean value to idicate if a term identifier exits
-   *    (TRUE) or of if it is a non-existent identifier (FALSE).
+   *     - 'is_saved': the expected return value of the method.
    *
-   * @dataProvider provideTermIdentifierForGetTermIdMethod
+   * @dataProvider provideTermsForSaveTermConfigValuesMethod
    */
-  public function testGetTermId($scenario, $input_term_identifier, $expected) {
-    $term_id = $this->service_PhenoTerms->getTermId($input_term_identifier);
-    $term_exists = ($term_id > 0) ? TRUE : FALSE;
+  public function testSaveTermConfigValuesMethod($scenario, $input_term, $term_identifier, $expected) {
+    // Create or fectch input term.
+    $term_exists = $this->chado_connection->select('1:cvterm', 'cvt')
+      ->fields('cvt', ['cvterm_id'])
+      ->condition('cvt.name', $input_term['name'])
```

Comment:
```suggestion
      ->condition('cvt.name', $input_term['name'], '=')
```
I added the comparison operator to make it easier to read.